### PR TITLE
Add Redux Devtool Enhancer as a developer dependency to the project

### PR DESCRIPTION
**Pull request by maxme**
`devToolsEnhancer` allows to
* Visualization of state, state history and state diff, and
* Dispatch actions from Chrome Redux devtools.

screen shot 2018-03-27 at 16 40 02

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,7 +9,8 @@
     "jest-expo": "25.0.0",
     "prettier": "git+https://github.com/Automattic/calypso-prettier.git#calypso-1.9",
     "react-native-scripts": "1.11.1",
-    "react-test-renderer": "16.2.0"
+    "react-test-renderer": "16.2.0",
+    "remote-redux-devtools": "^0.5.12"
```
Comment (hypest) [inline package.json]: Not sure how this works but, if it's a dev only dep, will importing it into `store/index.js` (as below) become broken in production?

Comment (maxme) [reply package.json]: Good point, I'll have to add a check and enable it for dev mode only.

Comment (hypest) [reply package.json]: https://github.com/zalmoxisus/redux-devtools-extension#14-using-in-production might be relevant.

Comment (diegoreymendez) [reply package.json]: Testing, will delete.  Please ignore.

Comment (maxme) [reply package.json]: Updated in 8692a96

Comment (hypest) [reply package.json]: Verified it doesn't load in production mode.

**Comment by hypest** [comment]
🛳️it

**Pull request merged**